src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  font-family: arial;
+  font-size: 13px;
+}
+
+.header-main {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.header-second,
+.header-link a {
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element
You can add `header-link` class for `<a>` element, instead of `<h2>`